### PR TITLE
SSL/TLSize all the things! (convert http:// to https:// where possible)

### PR DESCRIPTION
**Pull request by reedloden**
I know the blog uses GitHub Pages and isn't accessible over https:// (except for https://rubygems.github.io), but it would be good to at least have the download links and other things that could use https:// to actually use https://.

I also standardized on one UPGRADING URL since there were two of them being used (sadly neither support https://).

### CODE REVIEW

**Comment by reedloden** [comment]
@dwradcliffe, another one as well. :)

File: _posts/2010-05-13-1.3.7-released.md [hunk 1]
```diff
@@ -7,10 +7,10 @@ author_email: nick@gemcutter.org
 
 NOTE:
 
-http://rubygems.org is now the default source for downloading gems.
+https://rubygems.org is now the default source for downloading gems.
```
Comment (dwradcliffe) [inline _posts/2010-05-13-1.3.7-released.md]: These links probably shouldn't change, since this was `http` at the time of this post (I think?).

File: _posts/2013-03-05-2.0.1-released.md [hunk 2]
```diff
@@ -31,6 +31,6 @@ _Bug fixes:_
 * Allow specification of mode for gem subdirectory creation.  Ruby bug #7713 by nobu
 * Fix tests when an 'a.rb' exists.  Ruby bug #7749 by nobu.
 
-[download]: http://rubygems.org/pages/download
-[upgrading]: http://docs.seattlerb.org/rubygems/UPGRADING_rdoc.html
+[download]: https://rubygems.org/pages/download
+[upgrading]: http://rubygems.rubyforge.org/rubygems-update/UPGRADING_rdoc.html
```
Comment (dwradcliffe) [inline _posts/2013-03-05-2.0.1-released.md]: Why this change?

Comment (dwradcliffe) [inline _posts/2013-03-05-2.0.1-released.md]: Oh nevermind, I see your note at the top. :+1:

**Comment by dwradcliffe** [comment]
Other than the one post, LGTM. (Travis failure is fixed on master, btw)

File: _posts/2010-05-13-1.3.7-released.md (hunk 1 above)
Comment (reedloden) [inline _posts/2010-05-13-1.3.7-released.md]: Good catch. Changed back.

**Comment by reedloden** [comment]
Fixed that post and rebased to current master.

**Comment by dwradcliffe** [comment]
:+1:
